scripts/20170426-col828/add_impact_trending_to_assets.sql

```diff
@@ -0,0 +1,15 @@
+-- Add per-asset columns containing 'impact' and 'trending' scores and percentiles. These values
+-- are periodically recalculated and used interally for query selection and ordering, but do not 
+-- surface through the API.
+
+ALTER TABLE assets ADD impact_percentile integer NOT NULL DEFAULT 0;
+ALTER TABLE assets ADD impact_score integer NOT NULL DEFAULT 0;
+ALTER TABLE assets ADD trending_percentile integer NOT NULL DEFAULT 0;
+ALTER TABLE assets ADD trending_score integer NOT NULL DEFAULT 0;
```

Comment:
@paulkerschen Fyi, I believe the four ADDs above can live in a single ALTER statement:
```
ALTER TABLE table ADD COLUMN col1 int, ADD COLUMN col2 int, ...;
```
Gives you benefit of all or nothing. But, this is a one time script so I don't see a need for change.